modules/bullet/shape_bullet.h

```diff
@@ -53,6 +53,10 @@ class ShapeBullet : public RIDBullet {
 	Map<ShapeOwnerBullet *, int> owners;
 	real_t margin = 0.04;
 
+	// Contains the default shape.
+	btCollisionShape *default_shape = nullptr;
+	btCollisionShape *old_default_shape = nullptr;
+
```

Comment:
If `old_default_shape` is only used inside the one function, it does not need to be a class variable.

Reply:
The `old_default_shape` is used to not destroy the old default shape during shape refresh.